Formula/cava.rb

```diff
@@ -0,0 +1,40 @@
+class Cava < Formula
+  desc "Console-based Audio Visualizer for ALSA"
+  homepage "https://github.com/karlstav/cava"
+  url "https://github.com/karlstav/cava/archive/0.7.4.tar.gz"
+  sha256 "fefd3cc04d41b03ca416630cafadbfda6c75e2ca0869da1f03963dcb13e1ecb7"
+  license "MIT"
+  head "https://github.com/karlstav/cava.git", branch: "master"
+
+  depends_on "automake"
+  depends_on "fftw"
+  depends_on "iniparser"
+  depends_on "libtool"
+  depends_on "portaudio"
+
+  uses_from_macos "ncurses"
+
+  def install
+    # force autogen.sh to look for and use our glibtoolize
+    inreplace "autogen.sh", "libtoolize", "glibtoolize"
+
+    system "./autogen.sh"
+    ENV.append "CFLAGS", "-g -O0"
+    system "./configure", *std_configure_args, "--disable-silent-rules"
```

Comment:
This seems to ignore the `--prefix` setting

Reply:
Can you specify why it seems to do so?
Line 21 is needed as stated above.
Line 22 is a stupid bug I have to remove. Sorry for that.
Line 23 is more or less the example from the Formula template.

I found an issue on the arm64 machines:
-L/usr/local/lib is not found. This is probably due to some hardcoded LDFLAGS in upstreams build routines that I have to fix. Can you confirm that?

Reply:
It seem to install into the right place though. It just really wants to still look in `/usr/local/lib` for some reason.